modules/renter.go

```diff
@@ -67,9 +67,20 @@ type DownloadInfo struct {
 	StartTime   time.Time `json:"starttime"`
 }
 
+// An Allowance dictates how much the Renter is allowed to spend in a given
+// period.
+type Allowance struct {
+	Funds  types.Currency    `json:"funds"`
+	Hosts  int               `json:"hosts"`
+	Period types.BlockHeight `json:"period"`
+}
```

Comment:
Might want to add an extra parameter - `RefreshWindow`, which will dictate how far in advance the renter will call the 'renew' function. So if Period is 12 months and 'refresh window' is 6 weeks, it'll call renew after 10 months and 2 weeks (so there are 6 weeks or less remaining)